docs/streams/architecture.html

```diff
@@ -148,7 +148,8 @@ <h3 class="anchor-heading"><a id="streams_architecture_recovery" class="anchor-l
     <p>
         In addition, Kafka Streams makes sure that the local state stores are robust to failures, too. For each state store, it maintains a replicated changelog Kafka topic in which it tracks any state updates.
         These changelog topics are partitioned as well so that each local state store instance, and hence the task accessing the store, has its own dedicated changelog topic partition.
-        <a href="/{{version}}/documentation/#compaction">Log compaction</a> is enabled on the changelog topics so that old data can be purged safely to prevent the topics from growing indefinitely.
+        <a href="/{{version}}/documentation/#compaction">Log compaction</a> is enabled on the changelog topics so that old data can be purged safely to prevent the topics from growing indefinitely
+        (Keep in mind that this behavior is different from a "delete" clean-up policy in that the most recent event associated with a key is always preserved).
```

Comment:
`in which the`? Also maybe remove the parentheses.

Reply:
@ex172000 actually the explanation that comes after is for _Log compaction_ behavior (the former in the sentence) and not _delete_
I will remove the parentheses